Tasks/Common/VstsAzureHelpers_/Utility.ps1

```diff
@@ -496,23 +496,21 @@ function Disconnect-AzureAndClearContext {
             Write-Verbose "Trying to disconnect from Azure and clear context"
 
             if (Get-Command -Name "Disconnect-AzureRmAccount" -ErrorAction "SilentlyContinue") {
-                Write-Host "##[command]Disconnect-AzureRmAccount"
-                $null = Disconnect-AzureRmAccount
+                Write-Host "##[command]Disconnect-AzureRmAccount -Scope Process"
+                $null = Disconnect-AzureRmAccount -Scope Process
             }
             elseif (Get-Command -Name "Remove-AzureRmAccount" -ErrorAction "SilentlyContinue") {
-                Write-Host "##[command]Remove-AzureRmAccount"
-                $null = Remove-AzureRmAccount
+                Write-Host "##[command]Remove-AzureRmAccount -Scope Process"
+                $null = Remove-AzureRmAccount -Scope Process
             }
             elseif (Get-Command -Name "Logout-AzureRmAccount" -ErrorAction "SilentlyContinue") {
-                Write-Host "##[command]Logout-AzureRmAccount"
-                $null = Logout-AzureRmAccount
+                Write-Host "##[command]Logout-AzureRmAccount -Scope Process"
+                $null = Logout-AzureRmAccount -Scope Process
             }
 
             if (Get-Command -Name "Clear-AzureRmContext" -ErrorAction "SilentlyContinue") {
                 Write-Host "##[command]Clear-AzureRmContext -Scope Process"
                 $null = Clear-AzureRmContext -Scope Process
-                Write-Host "##[command]Clear-AzureRmContext -Scope CurrentUser -Force -ErrorAction SilentlyContinue"
```

Comment:
We did this recently in order so that contexts cannot be used across tasks or across jobs. Can you test that scenario once. Create a build/release with a AzurePs task and a subsequent powershell task and try executing any Azure Ps command in the powershell task. The command should fail. These changes look fine to me.

Reply:
thanks, Arjun,
Verified the scenario and the later Powershell task fails as expected.